_security/configuration/tls.md

```diff
@@ -264,15 +264,39 @@ These settings allow for the use of encrypted passwords in the settings.
 
 ## Hot reloading TLS certificates
 
-Updating expired or nearly expired TLS certificates does not require restarting the cluster. Instead, enable hot reloading of TLS cerificates by adding the following line to `opensearch.yml`:
+Updating expired or nearly expired TLS certificates on the HTTP and transport layers does not require restarting the cluster. Instead, you can enable hot reloading of TLS certificates using one of the following methods:
 
+- [Enabling the in-place hot reload setting](#enabling-the-in-place-hot-reload-setting)
+- Using the [Reload Certificates API](#using-the-reload-certificates-api)
 
-`plugins.security.ssl_cert_reload_enabled: true`
+If you use PEM files to manage certificates, your certificate and keys are stored in seperate files. Therefore, we recommend using rolling updates when updating reload settings to prevent authentication issues.
+{: .warning}
+
+### Enabling the in-place hot reload setting
+
+When enabled, in-place hot reloading monitors your keystore resources for updates every five seconds. If you add or modify a certificate, key file, or keystore setting in the Opensearch `config` directory, the nodes in the cluster detects the changes and reloads the keys and certificates.
+
+To enable in-place hot reloading, add the following line to `opensearch.yml`:
+
+```yml
+plugins.security.ssl.certificates_hot_reload.enabled: true
+```
+{% include copy.html %}
+
+### Using the Reload Certificates API
+
+Add the following to enable hot reloading using the Reload Certificates API:
+
+```yml
+plugins.security.ssl_cert_reload_enabled: true
+```
+{% include copy.html %}
 
 This setting is `false` by default.
 {: .note }
 
 After enabling hot reloading, use the Reload Certificates API to replace the expired certificates. The new certificates need to be stored in the same location as the previous certificates in order to prevent any changes to the `opensearch.yml` file. 
```

Comment:
```suggestion
After enabling reloading, use the Reload Certificates API to replace the expired certificates. The new certificates need to be stored in the same location as the previous certificates in order to prevent any changes to the `opensearch.yml` file. 
```

I think we should only use the term `hot reloading` for the new setting being introduced. It looks like its being used in both scenarios which can cause confusion.